common/queue/src/main/proto/queue.proto

```diff
@@ -66,11 +66,12 @@ enum SessionType {
 }
 
 enum KeyValueType {
-  BOOLEAN_V = 0;
-  LONG_V = 1;
-  DOUBLE_V = 2;
-  STRING_V = 3;
-  JSON_V = 4;
+  UNKNOWN = 0;
```

Comment:
This may cause us problems during upgrade (if some data is in the queue). Why not UNKNOWN= something else?